registry/metrics.go

```diff
@@ -37,20 +36,13 @@ func NewMetrics() Metrics {
 			Name:      "fetch_duration_seconds",
 			Help:      "Duration of Image metadata fetches, in seconds.",
 			Buckets:   stdprometheus.DefBuckets,
-		}, []string{fluxmetrics.LabelInstanceID, LabelRepository, fluxmetrics.LabelSuccess}),
+		}, []string{LabelRepositoryHost, fluxmetrics.LabelSuccess}),
```

Comment:
I was debating that. at the moment it's things like `index.docker.io`, `quay.io`, and `gcr.whatever`.
🤔 
I suppose if users are using self-hosted registries, it could be quite a few. Will disable for now until we know more.